### DISCUSSION TITLE
Why form submission inside a modal causes another form validation?

### DISCUSSION THREAD

**Discussion by moroshko** [react-hook-form/react-hook-form#6801] [Q&A] [answered]
I have 2 forms that are **not nested**, they live in sibling elements, one of them is inside a modal.
When the form inside the modal is submitted, for some reason the other form is submitted as well (triggering its validation) - this is a very unexpected and unwanted behavior.

See: [CodeSandbox](https://codesandbox.io/s/crazy-matan-ic15g)

To reproduce:
1. Click "Open modal"
2. Click Submit in the Modal
3. Observe that the first name input in the background is validated.

**Comment by bluebill1049** [comment]
avoid wrap form with form to avoid event bubble to the parent form

`<form><form/></form>`
to
`<form><form/>`

**Reply by moroshko** [reply]
Sorry, I don't understand. I explicitly stated that the forms are not nested.

**Reply by bluebill1049** [reply]
<img width="426" alt="Screen Shot 2021-10-14 at 11 26 37 am" src="https://user-images.githubusercontent.com/10513364/137230556-2737c18f-acea-4c1b-9f5e-1665344d4b22.png">

react portal will still propagate event

**Reply by moroshko** [reply]
@bluebill1049 I must be missing something. Event propagation only goes up to the parents, right? How does it get to the other sibling form? The DOM looks like this:
```
div
  form1 // how does the submit event get here?
portal
  form2 // submitted
```

**Reply by Jamalahmad123** [reply]
Hi @moroshko, Did you solved this issue. I'm also facing the exact same issue

**Comment by bluebill1049** [comment, marked as answer]
https://github.com/react-hook-form/react-hook-form/discussions/3704#discussioncomment-265857

**Reply by ParichayDidwania** [reply]
@bluebill1049  I am experiencing this issue in a different case. Both my button and modal component are outside the form, but the onSubmit inside the modal form is still triggering the sibling form.

This is the order:

Modal
form (open and close)
button to showModal
